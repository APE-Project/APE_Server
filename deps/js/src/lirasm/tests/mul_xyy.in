; 46340 * 46340 < 2^31, and will not overflow.
big = immi 46340

res = mulxovi big big    ; no overflow, so we don't exit here

; Ensure that 'big' gets its own register and isn't shared with 'res'.
; Also store 'res' so it isn't dead.
m = allocp 8
sti big m 0
sti res m 4
x                       ; we exit here
