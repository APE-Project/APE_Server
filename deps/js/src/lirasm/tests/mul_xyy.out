Exited block on line: 11
